test/e2e/chaos/pod.go

```diff
@@ -184,6 +186,118 @@ var _ = ginkgo.Describe("[chaos-mesh] Basic", func() {
 		framework.ExpectNoError(err, "Pod kill chaos perform failed")
 		cancel()
 	})
+
+	ginkgo.It("PauseChaos", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		nd := fixture.NewCommonNginxDeployment("nginx", ns, 3)
+		_, err := kubeCli.AppsV1().Deployments(ns).Create(nd)
+		framework.ExpectNoError(err, "create nginx deployment error")
+		err = waitDeploymentReady("nginx", ns, kubeCli)
+		framework.ExpectNoError(err, "wait nginx deployment ready error")
+
+		var pods *corev1.PodList
+		listOption := metav1.ListOptions{
+			LabelSelector: labels.SelectorFromSet(map[string]string{
+				"app": "nginx",
+			}).String(),
+		}
+		pods, err = kubeCli.CoreV1().Pods(ns).List(listOption)
+		framework.ExpectNoError(err, "get nginx pods error")
+
+		uids := getUIDs3Pods(pods.Items)
+
+		podKillChaos := &v1alpha1.PodChaos{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "nginx-kill",
+				Namespace: ns,
+			},
+			Spec: v1alpha1.PodChaosSpec{
+				Selector: v1alpha1.SelectorSpec{
+					Namespaces:     []string{ns},
+					LabelSelectors: map[string]string{"app": "nginx"},
+				},
+				Action: v1alpha1.PodKillAction,
```

Comment:
I think `pod-failure` is more available for this case, we can check the status of all pods to judge wheather the `pause`  is work.

Reply:
we would finally cover all kinds of chaos in every case in e2e, "schedule / duration / pause", current e2e test is ok.